gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -78,25 +80,34 @@
 @AndroidEntryPoint
 public class HomeScreenFragment extends AbstractFragment
     implements BackPressListener, OnNavigationItemSelectedListener, OnGlobalLayoutListener {
+
   // TODO: It's not obvious which feature are in HomeScreen vs MapContainer; make this more
   // intuitive.
   private static final float COLLAPSED_MAP_ASPECT_RATIO = 3.0f / 2.0f;
 
-  @Inject AddFeatureDialogFragment addFeatureDialogFragment;
-  @Inject AuthenticationManager authenticationManager;
-  @Inject Schedulers schedulers;
-  @Inject Navigator navigator;
-  @Inject EphemeralPopups popups;
+  @Inject
+  AddFeatureDialogFragment addFeatureDialogFragment;
+  @Inject
+  AuthenticationManager authenticationManager;
+  @Inject
+  Schedulers schedulers;
+  @Inject
+  Navigator navigator;
+  @Inject
+  EphemeralPopups popups;
   MapContainerViewModel mapContainerViewModel;
 
-  @Nullable private ProgressDialog progressDialog;
+  @Nullable
+  private ProgressDialog progressDialog;
```

Comment:
don't do stylistic changes that are not consistent with the project.

Reply:
If you install `google-java-format` in Android Studio the correct format can be applied automatically (optionally on save).